### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by manjanac**
Fixes:

1. Update model path check to use `standardizedFileURL`.
2. Add a `downloadStatus` property to prevent multiple concurrent downloads.

### CODE REVIEW

File: FirebaseMLModelDownloader/Sources/ModelDownloadTask.swift [hunk 1]
```diff
@@ -43,7 +42,7 @@ class ModelDownloadTask: NSObject {
   /// User defaults to which local model info should ultimately be written.
   private let defaults: UserDefaults
   /// Keeps track of download associated with this model download task.
-  private(set) var downloadStatus: ModelDownloadStatus
+  private(set) var downloadStatus: ModelDownloadStatus = .complete
```
Comment (maksymmalyhin) [inline FirebaseMLModelDownloader/Sources/ModelDownloadTask.swift:45]: Why `.complete` is the default state?

File: FirebaseMLModelDownloader/Sources/ModelDownloadTask.swift [hunk 2]
```diff
@@ -59,19 +58,19 @@ class ModelDownloadTask: NSObject {
...
   /// Name for model file stored on device.
   var downloadedModelFileName: String {
-    return "fbml_model__\(appName)__\(remoteModelInfo.name)"
+    return "fbml_model__\(appName)__\(remoteModelInfo.name).tflite"
   }
 
   func download(progressHandler: ProgressHandler?, completion: @escaping Completion) {
     /// Prevent multiple concurrent downloads.
-    guard downloadStatus == .notStarted else {
+    // TODO: Merge if there are multiple same requests.
```
Comment (maksymmalyhin) [inline FirebaseMLModelDownloader/Sources/ModelDownloadTask.swift:72]: I think merging should happen on the class user level.

File: FirebaseMLModelDownloader/Sources/ModelDownloadTask.swift (hunk 1 above)
Comment (manjanac) [reply FirebaseMLModelDownloader/Sources/ModelDownloadTask.swift:45]: Ah I reduced the possible statuses to be either `complete` or `inProgress`. Earlier, I had a `notStarted` as well but it seemed a bit redundant. I can bring it back for clarity though.

File: FirebaseMLModelDownloader/Sources/ModelDownloadTask.swift (hunk 2 above)
Comment (manjanac) [reply FirebaseMLModelDownloader/Sources/ModelDownloadTask.swift:72]: Do you mean at the `ModelDownloader` level?

File: FirebaseMLModelDownloader/Sources/ModelDownloadTask.swift (hunk 1 above)
Comment (maksymmalyhin) [reply FirebaseMLModelDownloader/Sources/ModelDownloadTask.swift:45]: It looks a bit confusing when initial status is `complete` to me. But I'm mostly fine with either way (not a blocker for the PR for me).

File: FirebaseMLModelDownloader/Sources/ModelDownloadTask.swift (hunk 2 above)
Comment (maksymmalyhin) [reply FirebaseMLModelDownloader/Sources/ModelDownloadTask.swift:72]: yes

**Review by maksymmalyhin — APPROVED** [review]
LGTM

File: FirebaseMLModelDownloader/Sources/ModelDownloadTask.swift (hunk 1 above)
Comment (manjanac) [reply FirebaseMLModelDownloader/Sources/ModelDownloadTask.swift]: Ok makes sense, I brought back `.notStarted` to clear this up.

File: FirebaseMLModelDownloader/Sources/ModelDownloadTask.swift (hunk 2 above)
Comment (manjanac) [reply FirebaseMLModelDownloader/Sources/ModelDownloadTask.swift]: Moved the TODO, thanks!

**Pull request merged**